Slider/Slider.js

```diff
@@ -106,6 +107,14 @@ const SliderBase = (props) => {
 		onWheel: handle(
 			forProp('disabled', false),
 			forwardWithPrevent('onWheel'),
+			(ev, {wheelEventInterval}) => {
+				if (ev.timeStamp - lastWheelEventTimeStamp.current < wheelEventInterval) {
+					ev.stopPropagation();
```

Comment:
Do we need to stop propagation here?

Reply:
It's fixed as this part moved to handleIncrementByWheel.